### ISSUE TITLE
Installation Error (unitttest.sh)

### ISSUE THREAD

**Issue by shaharyarahmad** [pkp/xmlps#90]
I get the following error when I run the unittest.sh

There was 1 error:

1) BibtexreferencesConversionTest\Model\Queue\BibtexreferencesJobTest::testConversion
Exception: Couldn't find the stage document

/var/www/html/module/BibtexreferencesConversion/src/BibtexreferencesConversion/Model/Queue/Job/BibtexreferencesJob.php:29
/var/www/html/module/BibtexreferencesConversion/test/BibtexreferencesConversionTest/Model/Queue/BibtexreferencesJobTest.php:47

FAILURES!
Tests: 31, Assertions: 95, Errors: 1.

**Comment by shaharyarahmad** [comment]
Wanted to update that the installation was made using the vagrant shell script. The current CERMINE is 1.11, this issue seems similar to the one posted by Vitaliy.

**Comment by Vitaliy-1** [comment]
And what the Open Typesetting Stack? What errors you see there?
I think that this error is not crucial. I have used manual installation - created Ubuntu 14 on virtual box to control every step.
Also I applied minor changes in composer.json to install all correctly. That was adding line 8 and 9: 
```
"minimum-stability": "dev",
"prefer-stable": true, 
```
 and change in line 92:

`"zendframework/zen-developer-tools": "1.*",`

After downloading cermine 1.11 there is a need to change global.php line about the cermine path

**Comment by shaharyarahmad** [comment]
I installed using the vagrant script, which included CERMINE 1.11 by default and had no need to download it myself. I installed all the dependencies automatically. 

I also agree the error might not be extremely fatal, but I am having an issue an issue with the citation style selector. I get the following error when trying to select citation styles.

Error loading CSL style file. Libxml error:
Start tag expected, '<' not found

I hope @axfelix might be able to help.
